lib/systems/src/AMSSytem.cpp

```diff
@@ -0,0 +1,121 @@
+#include "AMSSystem.h"
+
+void AMSSystem::init(unsigned long curr_micros) {
+    _last_heartbeat_time = curr_micros;
+}
+
+void AMSSystem::update_ams_system(unsigned long curr_micros, BMSData_s &bms_data, VCRSystemData_s &system_data) {
```

Comment:
I changed the output to void. This data instead takes in a reference to the global system data and manually updates those values.